tests/codebuild/common.sh

```diff
@@ -33,7 +33,7 @@ function wait_for_crd_status_else_fail()
 
```

Comment:
Can we rename the function, now that it doesn't `exit`? `wait_for_crd_status` I think is apt.